CHANGELOG.rst

```diff
@@ -8,6 +8,17 @@ inverse chronological order.
 Changes since version 5.0
 -------------------------
 
+- Change from stickybar to bottombar
+
+  For :func:`cli.run` to generate a status bar, it now requires the external
```

Comment:
The reference should be `nutils.cli.run`.

Reply:
Fixed.